src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -2960,25 +2960,27 @@ type AdaptiveFSharpLspServer
 
     override x.Dispose() = disposables.Dispose()
 
-    member this.WorkDoneProgressCancel(token: ProgressToken) : Async<unit> =
+    member this.WorkDoneProgressCancel(param: WorkDoneProgressCancelParams) : Async<unit> =
       async {
 
-        let tags = [ "ProgressToken", box token ]
+        let tags = [ "WorkDoneProgressCancelParams", box param ]
```

Comment:
What does the wrapper look like when used as an OTel tag? Should we just use the token here like we do in logging below?

Reply:
Things generally are just stringified, so in this case:

![image](https://github.com/ionide/FsAutoComplete/assets/1490044/ff8bb666-65d0-4a21-87dc-2930c9bf0121)